{{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/urls.py

```diff
@@ -21,6 +21,7 @@
     path('accounts/', include('allauth.urls')),
     path('api/v1/', include('home.api.v1.urls')),
     path('admin/', admin.site.urls),
+    path("users/", include("users.urls", namespace="users")),
```

Comment:
Isn't this scaffold on django 2, this will give error. it asks for app_name as well.

Reply:
It works. I have a test app running here http://django-api-test1-dev-811.botics.co/

The repo: https://github.com/crowdbotics-apps/django-api-test1-dev-811

Reply:
Well as long as it works. I had experienced error with this on django 2.